api/app/src/providers/withings.ts

```diff
@@ -105,16 +106,24 @@ export class Withings extends Provider implements OAuth2 {
     ].flat();
 
     for (const category of webhookCategories) {
-      const resp = await axios.post(
-        "https://wbsapi.withings.net/notify",
-        `action=subscribe&callbackurl=${callbackUrl}/&appli=${category}`,
-        {
-          headers: {
-            Authorization: `Bearer ${access_token}`,
-          },
-        }
-      );
-      console.log(resp.data);
+      try {
+        const resp = await axios.post(
+          "https://wbsapi.withings.net/notify",
+          `action=subscribe&callbackurl=${callbackUrl}/&appli=${category}`,
+          {
+            headers: {
+              Authorization: `Bearer ${access_token}`,
+            },
+          }
+        );
+        console.log(resp.data);
+      } catch (error) {
+        capture.error(error, {
+          extra: { context: `withings.postAuth` },
```

Comment:
can we send the URL and the body as well?

Reply:
There is no body, I can send the url though

Reply:
Nvm will send both

Reply:
nvm again - I think we were doing the sub incorrectly in the first place